Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -0,0 +1,124 @@
+
+  private lazy var amountInputView: AmountInputView = { AmountInputView(frame: .zero) }()
+  private lazy var inputStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var label: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var stepper: UIStepper = { UIStepper(frame: .zero) }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    _ = self
+      |> \.accessibilityElements .~ [self.label, self.stepper, self.amountInputView]
+      |> \.backgroundColor .~ UIColor.hex(0xf0f0f0) // Extract to Colors.json
+
+    self.contentView.addSubviewConstrainedToEdges(self.rootStackView)
+
+    self.rootStackView.addArrangedSubview(self.label)
+    self.rootStackView.addArrangedSubview(self.inputStackView)
+    self.inputStackView.addArrangedSubview(self.stepper)
+    self.inputStackView.addArrangedSubview(UIView(frame: .zero))
+    self.inputStackView.addArrangedSubview(self.amountInputView)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+    super.traitCollectionDidChange(previousTraitCollection)
+
+    let traitCollection = self.traitCollection
+
+    if previousTraitCollection?.preferredContentSizeCategory
+      != traitCollection.preferredContentSizeCategory {
+      self.configureView(for: traitCollection)
+    }
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.inputStackView
+      |> inputStackViewStyle
+
+    _ = self.label
+      |> labelStyle
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.stepper
+      |> stepperStyle
+  }
+
+  // MARK: - Configuration
+
+  private func configureView(for traitCollection: UITraitCollection) {
+    let isAccessibilityCategory = self.traitCollection.ksr_isAccessibilityCategory()
+
+    let alignment: UIStackView.Alignment = (isAccessibilityCategory ? .leading : .center)
+    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
+    let distribution: UIStackView.Distribution = (isAccessibilityCategory ? .equalSpacing : .fill)
+
+    _ = self.inputStackView
+      |> \.alignment .~ alignment
+      |> \.axis .~ axis
+      |> \.distribution .~ distribution
+      |> \.spacing .~ (isAccessibilityCategory ? Styles.grid(1) : 0)
+  }
+
+  func configureWith(value: (amount: Double, currency: String)) {
+    self.amountInputView.configureWith(
+      amount: String(format: "%i", value.amount),
+      placeholder: "\(0)",
+      currency: value.currency
+    )
+  }
+}
+
+// MARK: - Styles
+
+private let inputStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
+  stackView
+    |> \.spacing .~ Styles.grid(1)
```

Comment:
Yeah actually it seems that this isn't consistent with the spacing in `func configureView(for traitCollection: UITraitCollection)`